README.md

```diff
@@ -254,37 +254,37 @@ Check out our [TensorFlow.js demo](https://github.com/mindee/doctr-tfjs-demo) to
 
 ### Docker container
 
-[We offer Docker container support for easy testing and deployment](https://github.com/mindee/doctr/pkgs/container/doctr).
+We offer Docker container support for easy testing and deployment. [Here are the available docker tags.](https://github.com/mindee/doctr/pkgs/container/doctr).
 
 #### Using GPU with docTR Docker Images
 
-The docTR Docker images are GPU-ready and based on CUDA `11.8`.
-However, to use GPU support with these Docker images, please ensure that Docker is configured to use your GPU.
+The docTR Docker images are GPU-ready and based on CUDA `12.2`. Make sure your host is **at least `12.2`**, otherwise Torch or TensorFlow won't be able to initialize the GPU.
```

Comment:
I think we should also mention that the `nvidia-container-toolkit`
on the host system is required (https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html)
otherwise also with correct setup driver / CUDA / cudnn  it wouldn't detect your host GPU from the container correctly

Reply:
Ah i was to fast xD is solved ^^